samples/openapi3/client/petstore/dart2/petstore_client_lib/lib/auth/http_bearer_auth.dart

```diff
@@ -21,19 +21,31 @@ class HttpBearerAuth implements Authentication {
 
   set accessToken(dynamic accessToken) {
     if (accessToken is! String && accessToken is! HttpBearerAuthProvider) {
-      throw ArgumentError('Type of Bearer accessToken should be a String or a String Function().');
+      throw ArgumentError('accessToken value must be either a String or a String Function().');
     }
     _accessToken = accessToken;
   }
 
   @override
   void applyToParams(List<QueryParam> queryParams, Map<String, String> headerParams) {
+    if (_accessToken == null) {
+      return;
+    }
+
+    String accessToken;
+
     if (_accessToken is String) {
-      headerParams['Authorization'] = 'Bearer $_accessToken';
+      accessToken = _accessToken;
     } else if (_accessToken is HttpBearerAuthProvider) {
-      headerParams['Authorization'] = 'Bearer ${_accessToken()}';
+      accessToken = _accessToken!();
     } else {
-      throw ArgumentError('Type of Bearer accessToken should be a String or a String Function().');
+      return;
+    }
+
+    accessToken = accessToken.trim();
```

Comment:
same here, this can't be trimmed as we would be modifying user input.

Reply:
If a user wants to send value -1, we don't want to change it to 0 or 1 just because it's user provided negative number and we don't like negative numbers? openapi should not modify or edit any values user provided.Same reason as before, we can't convert null to empty string just because we would like it so. User wants to send null, null it is, user wants to send `\n\n\r\t` (EOL, EOL windows, tab) as username, we send it this way. if user provides `{password="password"}` as password, this clearly looks like JSON, but we have no business decoding it. generated code must act as a dumb actor and not modify or edit provided data.

Reply:
:heavy_check_mark: